civ13.dme

```diff
@@ -1077,5 +1077,5 @@
 #include "code\processes\lighting\sources.dm"
 #include "interface\interface.dm"
 #include "interface\skin.dmf"
-#include "maps\1943\smallingrad.dmm"
+#include "maps\nomads\nomads_africa.dmm"
```

Comment:
You've included your .dme file; Recompile with the smallingrand map and repush commit.